README.md

```diff
@@ -10,9 +10,22 @@ configure your local development settings.
 `cp config/config.dev.json.example config/config.dev.json`
 
 Also ensure you have yarn installed by following recommended install instructions:
-
 https://yarnpkg.com/en/docs/install#centos-stable
 
+Install development/build dependencies
+`yarn`
+
+Start the development server
```

Comment:
This would be a good time to update this. The DEVMODE=local is depricated & should be purged from the codebase. Issue to be filed. All local development is run with yarn start:remote

Reply:
We could also add this section now that local build testing is merged:

```
Test the production webpack bundle locally:
yarn start:remote:local-prod-test